documents/knowledge/elements.yaml

```
@@ -920,12 +920,18 @@
   description: Reduce a by b, or reverse each item of b
   arity: 2
   overloads:
+      num-num: a in base b, using a default alphabet 0-9A-Za-z
```

```suggestion
      num-num: a in base b, using a default alphabet 0-9A-Z
```
It's not using lowercase chars